README.md

```diff
@@ -1,4 +1,34 @@
 # ubiquity_motor
 [![Build Status](https://travis-ci.org/UbiquityRobotics/ubiquity_motor.svg?branch=indigo-devel)](https://travis-ci.org/UbiquityRobotics/ubiquity_motor)
 
-Package that provides a ROS interface for the motors in UbiquityRobotics robots
+Introduction
+
+This is a Package that provides a ROS interface for the motors in UbiquityRobotics robots
```

Comment:
Nit: Package should not be capitalized, and Ubiquity Robotics should be two words in text.